cmd/kubelet/app/server.go

```diff
@@ -244,17 +232,15 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 						return kubeletConfigFlagPrecedence(kc, args)
 					})
 				if err != nil {
-					klog.ErrorS(err, "Failed to bootstrap a configuration controller", "dynamicConfigDir", dynamicConfigDir)
-					os.Exit(1)
+					return fmt.Errorf("failed to bootstrap a configuration controller, dynamicConfigDir: %s", dynamicConfigDir)
```

Comment:
`err` is not getting wrapped here and `dynamicConfigDir` looks odd when embedded inside an error message like this.

Reply:
sorry, missed the error msg, Done:)